sphinx_automodapi/utils.py

```diff
@@ -13,6 +13,7 @@
            'find_autosummary_in_lines_for_automodsumm']
 
 SPHINX_LT_9 = Version(sphinx.__version__) < Version("9.0")
+SPHINX_LT_10 = Version(sphinx.__version__) < Version("10.0.dev")
```

Comment:
We started seeing this warning with the sphinx 9 release, so I suppose the patch should look for `LT_9`?